sources/packages/web/src/services/http/StudentAppealApi.ts

```diff
@@ -45,4 +49,19 @@ export class StudentAppealApi extends HttpBaseClient {
       this.handleAPICustomError(error);
     }
   }
+
+  /**
+   * Gets all pending student application exceptions.
+   * @param paginationOptions options to execute the pagination.
+   * @returns list of student application exceptions.
+   */
+  async getPendingAppeals(
+    paginationOptions: PaginationOptions,
+  ): Promise<PaginatedResultsAPIOutDTO<StudentAppealPendingSummaryAPIOutDTO>> {
+    let url = "appeal?";
+    url += getPaginationQueryString(paginationOptions);
+    return this.getCallTyped<
+      PaginatedResults<StudentAppealPendingSummaryAPIOutDTO>
```

Comment:
Please use the same `PaginatedResults` from the method return type (`PaginatedResultsAPIOutDTO`).

Reply:
I will update.